src/main/dml/fenixedu-academic.dml

```diff
@@ -5090,5 +5090,21 @@ class dml.DynamicField {
 		dml.DynamicField playsRole dynamicField { multiplicity *; }
 		.org.fenixedu.academic.domain.student.Registration playsRole registration { multiplicity 0..1; }
 	}
+	
+class dml.DynamicFieldTag {
+	String code (REQUIRED);
+	LocalizedString name (REQUIRED);
+	String domainObjectClassName (REQUIRED);
+}
+	
+	relation DynamicFieldTagRoot {
+		.org.fenixedu.bennu.core.domain.Bennu playsRole root { multiplicity 1..1; }
+		dml.DynamicFieldTag playsRole dynamicFieldTag { multiplicity *; }
```

Comment:
since this has a multiplicity of "many", the role name should be in plural (_dynamicFieldTags_)